.github/workflows/integration-tests.yml

```
@@ -109,6 +110,11 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: "3.11"
+      - name: Setup Dart
+        if: matrix.language == 'dart'
+        uses: dart-lang/setup-dart@v1.6.5
```

```suggestion
        uses: dart-lang/setup-dart@v1
```

So we don't get lots of dependabot PRs for minor/patch. GHA roll forward the major numbers